### ISSUE TITLE
Hitting the "Collapse" button (the one with 4 arrows pointing towards the center) on a wide display, crushes posts together. Example:

### ISSUE THREAD

**Issue by alecmuffett** [akirk/friends#174]
Example from stacking a few collapses on Safari/Mac:

<img width="381" alt="Screenshot 2023-01-08 at 10 11 19" src="https://user-images.githubusercontent.com/798224/211190588-4f5af808-8f21-48ae-a84d-07b49a183466.png">

**Comment by alecmuffett** [comment]
Aside: this was also very confusing on Mobile as a new user, because I was not sure if the `Reaction` button related to the post above or the post below, and having controls for each so close together on a mobile screen was hard to tap without doing the wrong thing.

Perhaps all the controls for a post could go in a single line below the post?

**Comment by akirk** [comment]
I agree that the collapsed styling of status posts is not good and needs to be improved. It's just a few CSS modifications:

https://github.com/akirk/friends/blob/4d86d25548dc51b3f1f61081add14629f1e37a70/templates/frontend/parts/content-status.php#L11

vs

https://github.com/akirk/friends/blob/4d86d25548dc51b3f1f61081add14629f1e37a70/templates/frontend/parts/content.php#L11

Having the `card` class removed makes statusses appear lighter by removing "card box" but then also causes these problems.

**Comment by akirk** [comment]
I improved it in 2.6.0 with #220 and am continuing to iterate with upcoming #224. Happy to get your thoughts on these improvements.
